cmd/base64_encoder/encoder.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+)
+
+var help = "The data field accepts a JSON or a string value. THE data flag must be set. Two example commands are shown below: \n\n" +
+	"  encoder -data='{\"userID\":\"admin\"}' \n" +
+	"  encoder -data='value' \n"
+
+func main() {
+	data := flag.String("data", "", "json or string data to be encoded. Surround the JSON data with single quotes. An example json data is '{\"userID\":\"admin\"}'")
```

Comment:
why use a flag if there is only one input?
do you plan or foresee the need to use additional flags in the future? maybe different encodings? i.e. base64std, vs base64url, etc?

Reply:
This is done for the consistency with other utilities such as `signer`, `bdb`.